README.md

```diff
@@ -1,4 +1,4 @@
-# Git Machete IntelliJ Plugin
+# Git Machete IntelliJ  Plugin
```

Comment:
A changes to the scenario descriptions are needed; and new recordings probably 😕

Reply:
At least the former! although the moment for the latter has also probably come... given the recent change in icons

Reply:
I will cover changes to README.md text in this one then. For the rest I have created: 
https://github.com/VirtusLab/git-machete-intellij-plugin/issues/850